velox/functions/sparksql/StringToMap.h

```diff
@@ -23,80 +23,111 @@ namespace facebook::velox::functions::sparksql {
 template <typename T>
```

Comment:
Would you update the description and add corresponding examples?
https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/string.rst?plain=1#L240-L252

Reply:
done